### DISCUSSION TITLE
Advanced: Dashboard Page for Users showing daily, AI-curated News Updates

### DISCUSSION THREAD

**Discussion by secondtruth** [mckaywrigley/chatbot-ui#1453] [Ideas]
I want to propose the development of an informational, user-customizable dashboard page, displaying current news about topics the user is interested in, summarized by the chosen model.

## Details

The data would be retrieved via RSS feeds and/or the [also proposed websearch feature](https://github.com/mckaywrigley/chatbot-ui/discussions/1359). The user can select interests and other important topics in their profile. The dashboard is regenerated automatically every day or upon user request. The user can ask question about the current dashboard contents, which turns it into a new chat session with the model.

Suggestions and improvements of this proposal are always very appreciated!

## Potential Future Enhancements
- Personalization of news summary length and complexity based on user preference.
- Option for users to select specific news sources or blacklist certain topics.
- Integration with calendar, to-do lists, notes, etc. for a comprehensive dashboard experience.
- Ability to archive and retrieve past dashboard summaries for reference.
- Feature to [share dashboard highlights internally](https://github.com/mckaywrigley/chatbot-ui/1513) or via social media or email.

## See also
- https://github.com/mckaywrigley/chatbot-ui/discussions/1485
- https://github.com/mckaywrigley/chatbot-ui/discussions/1513

**Comment by haydenkong** [comment]
This is great!
